flights/helpers/decorators.py

```diff
@@ -0,0 +1,11 @@
+from threading import Thread
+from functools import wraps
+
+
+def start_backgroun_job(func):
```

Comment:
Let's give this function a doc-string. 
I recommend adding an additional tool to help with this; `pip install flake8-docstrings` (https://pypi.org/project/flake8-docstrings/).

Run one last `flake8` validation for the project and fix any reported voilations.

Reply:
all done